csclient/csclient.go

```diff
@@ -174,37 +177,42 @@ func (c *Client) GetArchive(id *charm.URL) (r io.ReadCloser, eid *charm.URL, has
 	}
 	resp, err := c.Do(req, u.String())
 	if err != nil {
-		return nil, nil, "", 0, errgo.NoteMask(err, "cannot get archive", isAPIError)
+		terr := params.MaybeTermsAgreementError(err)
+		if err1, ok := errgo.Cause(terr).(*params.TermAgreementRequiredError); ok {
+			terms := strings.Join(err1.Terms, " ")
+			return fail(errgo.Newf(`cannot get archive because some terms have not been agreed to. Try "juju agree %s"`, terms))
```

Comment:
`Declined: please agree to the following terms %v. Try: "juju agree %v"`, terms, terms

That keeps it the same as juju